samza-azure/src/main/java/org/apache/samza/system/eventhub/consumer/EventHubSystemConsumer.java

```diff
@@ -306,7 +343,7 @@ public void start() {
 
   private void renewPartitionReceiver(SystemStreamPartition ssp) {
     String streamId = config.getStreamId(ssp.getStream());
-    EventHubClientManager eventHubClientManager = streamEventHubManagers.get(streamId);
+    EventHubClientManager eventHubClientManager = perPartitionEventHubManagers.get(ssp);
```

Comment:
What happens when the connection is lost? Don't we need to renew the EventHubClient to create a new connection?

Reply:
Temporary connection lost doesn't need a new EventHubClient. I have tested the behavior.